KesaApplication/LoginViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  LoginViewController.swift
+//  KesaApplication
+//
+//  Created by Jiwoo Lim on 2016. 6. 1..
+//  Copyright © 2016년 kesa. All rights reserved.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController {
+    
+    //Function to display error message
+    func displayAlertMessage(userMessage: String){
+        let myAlert = UIAlertController(title: "Error", message: userMessage, preferredStyle: UIAlertControllerStyle.Alert);
+        
+        let okAction = UIAlertAction(title: "OK", style: UIAlertActionStyle.Default, handler:nil);
+        
+        myAlert.addAction(okAction);
+        
+        self.presentViewController(myAlert, animated: true, completion: nil);
+    }
+
+    //Two text fields
+    @IBOutlet weak var userEmailTextField: UITextField!
+    @IBOutlet weak var userPasswordTextField: UITextField!
+    
+    @IBAction func loginButtonTapped(sender: AnyObject) {
```

Comment:
I think **onLoginButtonClickListener** is a better name for this function. Tapped is not clear enough. It is kind of ambiguous to me.